poker/utils/message.py

```diff
@@ -0,0 +1,148 @@
+        if player_response in [Decision.Y.value, Decision.YES.value]:
+            return True
+        return False    
+    
+    def starting_cash(self) -> int:
+        player_response = int(input(f"How much money would you like to start off with? {self.cash_options} "))
+        if player_response not in self.cash_options:
+            raise CashException
+        return player_response
+    
+    def competition_count(self) -> int:
+        count = int(input(f"How many players would you like to play against? {COMPETITION} "))
+        if count not in COMPETITION:
+            raise RangeException
+        return count
+
+    def competition_cash(self) -> int:
+        cash = int(input(f"How much money should each player get? {self.cash_options[-2::]} "))
+        if cash not in self.cash_options:
+            raise CashException
+        return cash
+    
+    def action(self, has_raise: bool, raise_amount: int) -> str:
+        if has_raise:
+            player_response = input(f"The bet was raised by {raise_amount}, what would you like to do? [{Action.CALL} or {Action.FOLD}] ").lower()
+            if player_response not in [Action.CALL, Action.FOLD]:
+                raise InvalidActionException
+        else:
+            player_response = input(f"What would you like to do? [{Action.CHECK}, {Action.RAISE} or {Action.FOLD}] ").lower()
+            if player_response not in [Action.CHECK, Action.RAISE, Action.FOLD]:
+                raise InvalidActionException
+        return player_response
+
+    def action_taken(self, name: str, action: str, amount: int, possible_actions: list[str] = []) -> None:
+        print(f"{name} decided to {action} {amount if action in possible_actions else ''}") 
+    
+    def increase(self, chip_count: int) -> int:
+        player_response = int(input(f"How much would you like to raise? "))
+        if player_response <= 0:
+            raise NegativeException
+        if player_response >= chip_count:
+            raise InsufficientChipException
+        return player_response
+    
+    def different_amount(self, chip_count: int) -> None:
+        print(f"You only have {chip_count} chips. Please select a lower amount.")
+    
+    def player_summary(self, players: dict) -> None:
+        player_table = Table(title="Player Summary")
+        player_table.add_column("Order")
+        player_table.add_column("Name")
+        player_table.add_column("Chips")
+        player_table.add_column("Blind")
+        player_table.add_column("Pocket Cards")
+        player_table.add_column("Best Hand")
+        player_table.add_column("Short Name")
+        for player_id, player in players.items():
+            player = player["player"]
+            player_table.add_row(
+                str(player_id), 
+                player.name, 
+                f"{GameStack.WHITE['name']}: {player.stack.chips[GameStack.WHITE['name']]}",
```

Comment:
this is a bit hard to read, can you move all this inline code into intermediate variables so the add_row becomes more concise?

Reply:
completely agree. I'll work on making it more readable/concise.